playwright/e2e/ui/pages/home.py

```diff
@@ -454,7 +480,7 @@ async def click_new_chapter(self):
 
     @pytest.mark.asyncio
     async def click_show_hide_solution_link(self):
-        await self.page.locator("id=fs-id1165134108429").get_by_title(
+        await self.page.locator("#fs-id1165134108429").get_by_title(
```

Comment:
I think these ids are book page-specific, it's probably not great to rely on them.